<ui version="4.0" >
 <class>SudokuForm</class>
 <widget class="QWidget" name="SudokuForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SudokuRapid</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGraphicsView" name="graphicsView" />
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QPushButton" name="newButton" >
        <property name="text" >
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton" >
        <property name="text" >
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="randomButton" >
        <property name="text" >
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="solveButton" >
        <property name="text" >
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton" >
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>SudokuForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>371</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel" >
     <x>215</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
